pkg/actuators/machine/actuator.go

```diff
@@ -79,13 +80,15 @@ func (a *Actuator) Create(ctx context.Context, machine *machinev1.Machine) error
 		awsClientBuilder: a.awsClientBuilder,
 	})
 	if err != nil {
-		return a.handleMachineError(machine, err, createEventAction)
+		fmtErr := fmt.Sprintf(scopeFailFmt, machine.GetName(), err)
+		return a.handleMachineError(machine, fmt.Errorf(fmtErr), createEventAction)
```

Comment:
isn't this making us lose the error type `machineapierros.InvalidMachineConfiguration` `MachineError`?

Reply:
we should have a unit test which validate that ^^